src/calculateRentalCost.js

```diff
@@ -4,7 +4,29 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAY_COST = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let totalAmount = 0;
```

Comment:
Since `totalAmount` is assigned a value only once in each conditional block and not modified afterwards, you can use `const` in each block instead of `let` at the top. This ensures that the variable is not unintentionally modified later on and adheres to checklist item 3, which prefers `const` over `let`.